packages/main/src/components/DynamicPage/index.tsx

```diff
@@ -1,3 +1,4 @@
+import { enrichEventWithDetails } from '@ui5/webcomponents-react-base/dist/Utils';
```

Comment:
This should import from `lib`:
```suggestion
import { enrichEventWithDetails } from '@ui5/webcomponents-react-base/lib/Utils';
```

Reply:
Done